back/Gemfile.lock

```diff
@@ -998,6 +990,8 @@ GEM
     socksify (1.7.1)
     spring (2.0.2)
       activesupport (>= 4.2)
+    spring-commands-rspec (1.0.4)
+      spring (>= 0.9.1)
```

Comment:
Unrelated, but you should share how you're using Spring for other Mac people who want to save some time :)